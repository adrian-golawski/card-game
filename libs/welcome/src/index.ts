export * from './lib/welcome.module';
export * from './lib/welcome-container/welcome-container.component';
