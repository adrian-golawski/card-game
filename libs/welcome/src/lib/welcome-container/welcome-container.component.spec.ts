import { WelcomeContainerComponent, WelcomeModule } from '@card-game/welcome';

import { Shallow } from 'shallow-render';

describe('WelcomeContainerComponent', () => {
  let shallow: Shallow<WelcomeContainerComponent>;

  beforeEach(() => {
    shallow = new Shallow(WelcomeContainerComponent, WelcomeModule);
  });

  it('should match the snapshot', async () => {
    const { fixture } = await shallow.render();

    expect(fixture).toMatchSnapshot();
  });
});
