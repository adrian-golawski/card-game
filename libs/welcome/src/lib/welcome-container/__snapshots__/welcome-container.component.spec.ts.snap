// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeContainerComponent should match the snapshot 1`] = `
<ng-component>
  <card-game-welcome-container>
    <p>
      welcome-container works!
    </p>
  </card-game-welcome-container>
</ng-component>
`;
